charts/alfresco-ai-transformer/templates/deployment-ai-transformer.yaml

```diff
@@ -20,8 +20,9 @@ spec:
     spec:
       serviceAccountName: {{ include "alfresco-ai-transformer.serviceAccountName" . }}
       {{- include "alfresco-common.component-pod-security-context" .Values | indent 4 }}
-      {{- if .Values.nodeSelector }}
-      nodeSelector: {{- .Values.nodeSelector | toYaml | nindent 8 }}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- .Values.nodeSelector | toYaml | nindent 8 }}
```

Comment:
move it to the bottom of the template together with toleration and affinity, like in https://github.com/Alfresco/alfresco-helm-charts/blob/main/charts/alfresco-search-enterprise/templates/liveindexing-deployment.yaml#L78 (I was following the same ordering of the helm boilerplate when you generate a new chart, should help when we make a new named template)

Reply:
I had to move the `affinity` value to a string to cope with default & comon uses cases